### ISSUE TITLE
Crash on placing ladder

### ISSUE THREAD

**Issue by Craluminum2413** [anegostudios/VintageStory-Issues#1025]
**Game Version:** 1.15.0-pre.4
**Platform:** Windows
**Modded:** No

### Description
Game crashes when trying to place a ladder on the surface. Also wooden ladder placing is very weird

### How to reproduce
1. Place the ladder right on top of the block, not on the wall
2. It crashes

### Videos

https://user-images.githubusercontent.com/69315569/122656533-230e3300-d164-11eb-9ef8-45a25fe1f3b3.mp4

### Logs
```
Running on 64 bit Windows with 16 GB RAM 
Version: v1.15.0-pre.4 (Unstable)06/20/2021 00:50:19: Critical error occurred
System.NullReferenceException: Object reference not set to an instance of an object.
   at Vintagestory.GameContent.BlockBehaviorLadder.HasSupport(Block forBlock, IBlockAccessor blockAccess, BlockPos pos)
   at Vintagestory.GameContent.BlockBehaviorLadder.TryPlaceBlock(IWorldAccessor world, IPlayer byPlayer, ItemStack itemstack, BlockSelection blockSel, EnumHandling& handled, String& failureCode)
   at Vintagestory.API.Common.Block.TryPlaceBlock(IWorldAccessor world, IPlayer byPlayer, ItemStack itemstack, BlockSelection blockSel, String& failureCode)
   at Vintagestory.Client.NoObf.ClientMain.OnPlayerTryPlace(BlockSelection blockSelection, String& failureCode)
   at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.OnBlockBuild(BlockSelection blockSelection, Block onBlock, String& failureCode)
   at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.HandleMouseInteractionsBlockSelected(Single dt)
   at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.OnFinalizeFrame(Single dt)
   at Vintagestory.Client.NoObf.ClientEventManager.TriggerRenderStage(EnumRenderStage stage, Single dt)
   at Vintagestory.Client.NoObf.ClientMain.TriggerRenderStage(EnumRenderStage stage, Single dt)
   at Vintagestory.Client.NoObf.ClientMain.RenderToDefaultFramebuffer(Single dt)
   at _nw5iq7Nu0RWJnxlBZen6mJGmcSN._MXRpcKwvmgsUnS5BPzRQ25DH3lh(Single )
   at _knXhXKQCYhvGbZXkDCncKaLldTL._BdeV9hzEfqFcm87gcxwrAtDeTgk(Single )
   at _knXhXKQCYhvGbZXkDCncKaLldTL._3JdxcXDlKfmdnBvyHIxr4AJFIze(Single )
   at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e)
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp)
   at OpenTK.GameWindow.DispatchRenderFrame()
   at OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second)
   at _SaXpqv5O911lCsSIrDnAnx1C2Wl._lESBMi7dCimysuVIyentZFkbZ2C(_3DvyrNaGS3KUgpwJWmWNJ0OhmVh , String[] )
   at _xYPi1FuSC15HmCPKeAQ55E3UGnQ._lESBMi7dCimysuVIyentZFkbZ2C(ThreadStart )
```

**Comment by Saricane** [comment]
Same with trying top place a ladder on top of a stone fence (which is probably the same as a surface liked described by Craluminum).

```
Running on 64 bit Windows with 16 GB RAM 
Version: v1.15.0-pre.4 (Unstable)20/06/2021 08:02:43: Critical error occurred
System.NullReferenceException: La référence d'objet n'est pas définie à une instance d'un objet.
   à Vintagestory.GameContent.BlockBehaviorLadder.HasSupport(Block forBlock, IBlockAccessor blockAccess, BlockPos pos)
   à Vintagestory.GameContent.BlockBehaviorLadder.TryPlaceBlock(IWorldAccessor world, IPlayer byPlayer, ItemStack itemstack, BlockSelection blockSel, EnumHandling& handled, String& failureCode)
   à Vintagestory.API.Common.Block.TryPlaceBlock(IWorldAccessor world, IPlayer byPlayer, ItemStack itemstack, BlockSelection blockSel, String& failureCode)
   à Vintagestory.Client.NoObf.ClientMain.OnPlayerTryPlace(BlockSelection blockSelection, String& failureCode)
   à Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.OnBlockBuild(BlockSelection blockSelection, Block onBlock, String& failureCode)
   à Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.HandleMouseInteractionsBlockSelected(Single dt)
   à Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.OnFinalizeFrame(Single dt)
   à Vintagestory.Client.NoObf.ClientEventManager.TriggerRenderStage(EnumRenderStage stage, Single dt)
   à Vintagestory.Client.NoObf.ClientMain.TriggerRenderStage(EnumRenderStage stage, Single dt)
   à Vintagestory.Client.NoObf.ClientMain.RenderToDefaultFramebuffer(Single dt)
   à _nw5iq7Nu0RWJnxlBZen6mJGmcSN._MXRpcKwvmgsUnS5BPzRQ25DH3lh(Single )
   à _knXhXKQCYhvGbZXkDCncKaLldTL._BdeV9hzEfqFcm87gcxwrAtDeTgk(Single )
   à _knXhXKQCYhvGbZXkDCncKaLldTL._3JdxcXDlKfmdnBvyHIxr4AJFIze(Single )
   à Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e)
   à System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   à OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp)
   à OpenTK.GameWindow.DispatchRenderFrame()
   à OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second)
   à _SaXpqv5O911lCsSIrDnAnx1C2Wl._lESBMi7dCimysuVIyentZFkbZ2C(_3DvyrNaGS3KUgpwJWmWNJ0OhmVh , String[] )
   à _xYPi1FuSC15HmCPKeAQ55E3UGnQ._lESBMi7dCimysuVIyentZFkbZ2C(ThreadStart )
-------------------------------
```

**Comment by Craluminum2413** [comment]
Fixed in 1.15.0-pre.6, I checked this in the game
> Game crashing when trying to place normal ladders
